app/_hub/kong-inc/rate-limiting-advanced/_metadata.yml

```
@@ -24,9 +24,7 @@ categories:
 publisher: Kong Inc.
 desc: Upgrades Kong Rate Limiting with more flexibility and higher performance
 description: |
-  The Rate Limiting Advanced plugin for Konnect Enterprise is a re-engineered version of the Kong Gateway (OSS) [Rate Limiting plugin](/hub/kong-inc/rate-limiting/).
-
-  As compared to the standard Rate Limiting plugin, Rate Limiting Advanced provides:
+  The Rate Limiting Advanced plugin for Konnect Enterprise offers more functionality than the Kong Gateway (OSS) [Rate Limiting plugin](/hub/kong-inc/rate-limiting/), such as:
```

```suggestion
Rate limit how many HTTP requests can be made in a given time frame.

 The Rate Limiting Advanced plugin offers more functionality than the Kong Gateway (OSS) [Rate Limiting plugin](/hub/kong-inc/rate-limiting/), such as:
```
Getting rid of "Konnect Enterprise" - it's referring to an old name for the Konnect Platform, which is a thing that doesn't exist.

Also adding a short one-liner description about what this plugin actually does.